src/content/docs/infrastructure/microsoft-azure-integrations/get-started/activate-azure-integrations.mdx

```diff
@@ -114,6 +116,53 @@ Follow these steps to rotate the Azure client secret in the Infratructure UI:
 2. Select the edit action next to Account Name to see and edit any application value.
 3. Edit the **Client Secret** field with the new value and confirm with **Save Changes**.
 
-## Explore app data in New Relic Infrastructure's UI [#find-data]
-
-After you activate an Azure integration, New Relic will start monitoring your Azure data at regular [polling intervals](/docs/infrastructure/microsoft-azure-integrations/azure-integrations-list/azure-polling-intervals-infrastructure-integrations). To [find and use your data](/docs/infrastructure/integrations/find-use-infrastructure-integration-data), use the [data explorer](/docs/query-your-data/explore-query-data/data-explorer/introduction-data-explorer) or go to [**one.newrelic.com**](https://one.newrelic.com) **> Infrastructure > Azure**
+## Uninstall Azure integrations [#uninstall]
+
+To uninstall Azure integrations, see the table below.
```

Comment:
"follow these steps" instead of "see the table below"?
This is just me. I don't like to refer to "above" or "below" in a doc because you never know when the info might get moved! :-O

Reply:
Thanks!